app/src/main/java/com/sun/weather/screen/home/HomeFragment.kt

```diff
@@ -107,6 +108,10 @@ class HomeFragment : BaseFragment<FragmentHomeBinding>(), HomeContract.View {
         updateUIWithCurrentWeather(currentWeather)
     }
 
+    override fun onSnackBar() {
+        Toast.makeText(requireContext(), "Thành phố đã được thêm vào yêu thích,không thể thêm nữa", Toast.LENGTH_SHORT).show()
```

Comment:
```suggestion
        Toast.makeText(requireContext(), "Thành phố đã được thêm vào yêu thích,không thể thêm nữa", Toast.LENGTH_SHORT).show()
```
Hỗ trợ thêm tiếng anh e nhé!